packages/integration-tests/src/tests/api/interaction/mfa/backup-code.test.ts

```diff
@@ -0,0 +1,111 @@
+import { InteractionEvent, MfaFactor, SignInIdentifier } from '@logto/schemas';
```

Comment:
nit: better add counter case that try to sign-in with a used backup code and fails.

Reply:
Good call, an issue added.